go/file_manager/pfsmodules/getfiles.go

```diff
@@ -0,0 +1,111 @@
+package pfsmodules
```

Comment:
Need better package naming.
Package name could be `pfs`, and there is a struct called `Server`, so when people using it, can do:
```go
import ".../pfs"
var server pfs.Server
```
It's easier to understand.

Please see: https://blog.golang.org/package-names